packages/redis/package.json

```diff
@@ -19,8 +19,9 @@
         "@walmartlabs/cookie-cutter-core": "^1.3.0-beta"
     },
     "devDependencies": {
+        "@types/redis": "2.8.16",
         "@walmartlabs/cookie-cutter-core": "^1.3.0-beta",
-        "@types/redis": "2.8.16"
+        "jest": "^26.0.1"
```

Comment:
no need to add this dependency ... it's part of the root workspace dev dependencies